hnn_core/viz.py

```diff
@@ -611,3 +682,76 @@ def plot_connectivity_matrix(conn, ax=None, show=True):
 
     plt_show(show)
     return ax.get_figure()
+
+
+def plot_cell_connectivity(net, conn_idx, src_gid, ax=None, show=True):
+    """Plot synaptic weight of connections from a src_gid.
+
+    Parameters
+    ----------
+    net : Instance of Network object
+        The Network object
+    conn_idx : int
+        Index of connection to be visualized
+        from `net.connectivity`
+    src_gid : int
+        Each cell in a network is uniquely identified by it's "global ID": GID.
```

Comment:
I'm in doubt what this function does. Could you be more explicit about the first two arguments? I don't understand which int's to enter. Loving the output though!!

Reply:
Definitely will add a better docstring. `conn_idx` is awkward to work with at the moment as it is really just the index of the connection you'd like to visualize in `net.connectivity[conn_idx]`. Unfortunately the alternative of passing the `_Connectivity` object directly isn't much better.

I think this awkwardness will be resolved when I implement a utility function discussed with @jasmainak. It will look like 
```py 
net.pick_connectivity(src_type='L5_pyramidal', receptor='ampa',...)
```
and return the indeces of all connections that match the provided arguments.